packages/design-tokens/src/scss/themes/theme-light/_colors.scss

```diff
@@ -0,0 +1,421 @@
+            default: $action-button-secondary-default,
+            hover: $action-button-secondary-hover,
+        ),
+        tertiary: (
+            active: $action-button-tertiary-active,
+            border: $action-button-tertiary-border,
+            content: $action-button-tertiary-content,
+            default: $action-button-tertiary-default,
+            hover: $action-button-tertiary-hover,
+        ),
+        plain: (
+            active: $action-button-plain-active,
+            border: $action-button-plain-border,
+            content: $action-button-plain-content,
+            default: $action-button-plain-default,
+            hover: $action-button-plain-hover,
+        ),
+    ),
+    link: (
+        primary: (
+            active: $action-link-primary-active,
+            default: $action-link-primary-default,
+            hover: $action-link-primary-hover,
+        ),
+        secondary: (
+            active: $action-link-secondary-active,
+            default: $action-link-secondary-default,
+            hover: $action-link-secondary-hover,
+        ),
+        tertiary: (
+            active: $action-link-tertiary-active,
+            default: $action-link-tertiary-default,
+            hover: $action-link-tertiary-hover,
+        ),
+        visited: (
+            default: $action-link-visited-default,
+        ),
+    ),
+    toggle: (
+        border-basic: $action-toggle-border-basic,
+        border-subtle: $action-toggle-border-subtle,
+        content-basic: $action-toggle-content-basic,
+        content-subtle: $action-toggle-content-subtle,
+        default-basic: $action-toggle-default-basic,
+        default-subtle: $action-toggle-default-subtle,
+        hover-basic: $action-toggle-hover-basic,
+        hover-subtle: $action-toggle-hover-subtle,
+    ),
+) !default;
+
+$background-colors: (
+    backdrop: $background-backdrop,
+    brand: (
+        primary: $background-brand-primary,
+        secondary: $background-brand-secondary,
+    ),
+    form-field: (
+        active: $background-form-field-active,
+        default: $background-form-field-default,
+        hover: $background-form-field-hover,
+    ),
+    interactive: (
+        active: $background-interactive-active,
+        default: $background-interactive-default,
+        hover: $background-interactive-hover,
+        selected: $background-interactive-selected,
+    ),
+    primary: $background-primary,
+    secondary: $background-secondary,
+    tertiary: $background-tertiary,
+) !default;
+
+$border-colors: (
+    interactive: (
+        active: $border-interactive-active,
+        danger: $border-interactive-danger,
+        default: $border-interactive-default,
+        hover: $border-interactive-hover,
+        informative: $border-interactive-informative,
+        selected: $border-interactive-selected,
+        success: $border-interactive-success,
+        warning: $border-interactive-warning,
+    ),
+    secondary: (
```

Comment:
No `primary` border?

Reply:
Good point, I opened a discussion with our designers and we can solve this one in another PR.